LibertyDSNP/project-liberty-web
# Feature/add scss to footer/177294936

Problem
=======
problem statement, including
[link to Pivotal Tracker #177294936](https://www.pivotaltracker.com/story/show/177294936)
[link to Pivotal Tracker #177310410](https://www.pivotaltracker.com/story/show/177310410)

Solution/Change summary
========
* take content from code of conduct page and scss and transfer it to this site
 * remove unused classes and universal styling elements
* add coc to App.js and footer
* Take styling from the original Project Liberty Website and apply it to the new one.

Steps to Verify:
----------------
1. run the site
2. make sure it looks as expected
